ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -544,19 +554,22 @@ private class JdbcLedgerDao(
       recordTime: Instant,
       offsetStep: OffsetStep,
       reason: RejectionReason,
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing rejection")
```

Comment:
Submitter info contains submissionId, please log it when provided

Reply:
Same comment as [above](https://github.com/digital-asset/daml/pull/8676#discussion_r571638038).